spannerlib/wrappers/spannerlib-ruby/spec/spannerlib/ruby_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+RSpec.describe Spannerlib::Ruby do
+  it "has a version number" do
+    expect(Spannerlib::Ruby::VERSION).not_to be nil
+  end
+
+  it "does something useful" do
+    expect(false).to eq(true)
+  end
```

Comment:
remove? I think that this is also proof that the tests are not being executed :-)